src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers.js';
+import {
+	createClass,
+	omitProps,
+	getFirst,
+} from '../../util/component-types.js';
+
+const cx = lucidClassNames.bind('&-ProgressBar');
+
+const { number, string, oneOf, node } = PropTypes;
+
+const ProgressBar = createClass({
+	displayName: 'ProgressBar',
+
+	statics: {
+		peek: {
+			description: `
+				A Progress Bar component is used to indicate progress in a procedure
+				consisting of numerous discrete steps or continuous operation.
+			`,
+
+			categories: ['communication'],
+		},
+	},
+
+	propName: 'ProgressBar',
+
+	components: {
+		Title: createClass({
+			displayName: 'ProgressBar.Title',
+			statics: {
+				peek: {
+					description: `
+						Content displayed at the top of the ProgressBar.
+					`,
+				},
+			},
+			propName: 'Title',
+		}),
+	},
+
+	propTypes: {
+		className: string`
+			Appended to the component-specific class names set on the root element.
+		`,
+
+		kind: oneOf(['default', 'success', 'danger', 'info', 'warning'])`
+			Applies a color style for the kind of ProgressBar.
+		`,
+
+		percentComplete: number`
```

Comment:
It's probably worth validating that this value is a valid percentage (i.e. a positive number between 0 and 100)

You could do that by using `oneOf` in combination with `_.range`

Something like this should work:
```
percentageComplete: oneOf(_.range(0, 101))
```

(this does assume that percentageComplete is an integer, and not a floating point value)